frontend/src/triggerAudioVisuals.js

```diff
@@ -5,7 +5,7 @@ import TIMEOUTS from './timeouts';
 const flashElement = (elementId) => {
   const element = document.getElementById(elementId.replace(/ *\([^)]*\) */g, ""));
   element.style.animation = "full-fade-in 1s forwards";
-  TIMEOUTS.setTimeout(() => {
```

Comment:
What was the purpose of removing the line out of interest? I would have thought that we would want it to clear any scheduled arrival effects when we restart?